nb-adapters/adapter-azure-aisearch/src/main/java/io/nosqlbench/adapter/azureaisearch/ops/AzureAISearchCreateOrUpdateIndexOp.java

```diff
@@ -28,16 +28,8 @@ public AzureAISearchCreateOrUpdateIndexOp(SearchIndexClient client, SearchIndex
 	public Object applyOp(long value) {
 		SearchIndex createResponse = null;
 		try {
-			if (logger.isDebugEnabled()) {
-				request.getFields().forEach((field) -> {
-					logger.debug(
-							">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>SearchIndex: Name:{}-ProfileName:{}-Type:{}-Dimension:{}",
-							field.getName(), field.getVectorSearchProfileName(), field.getType().toString(),
-							field.getVectorSearchDimensions());
-				});
-			}
 			createResponse = searchIndexClient.createOrUpdateIndex(request);
-			logger.debug("Successfully created the collection with return code of {}", createResponse.toString());
+			logger.debug("Successfully created the collection with return response: {}", createResponse.toString());
```

Comment:
not a big deal but the preferred form is e.g. NBCLI.logger.debug(() -> "user requests to cat " + resourceToCat);

Reply:
sometimes, I get into `local variables referenced from a lambda expression must be final or effectively final` situatation, but I updated them all now.